src/display/lv_display.h

```diff
@@ -231,10 +231,15 @@ int32_t lv_display_get_dpi(const lv_display_t * disp);
  * @param disp              pointer to a display
  * @param buf1              first buffer
  * @param buf2              second buffer (can be `NULL`)
+ */
+void lv_display_set_draw_buffers(lv_display_t * disp, lv_draw_buf_t * buf1, lv_draw_buf_t * buf2);
+
+/**
+ * Set display render mode
+ * @param disp              pointer to a display
  * @param render_mode       LV_DISPLAY_RENDER_MODE_PARTIAL/DIRECT/FULL
  */
-void lv_display_set_draw_buffers(lv_display_t * disp, void * buf1, void * buf2, uint32_t buf_size_in_bytes,
-                                 lv_display_render_mode_t render_mode);
+void lv_display_set_render_mode(lv_display_t * disp, lv_display_render_mode_t render_mode);
```

Comment:
Why do we need a separate function for this? I think the rendering mode is coupled with the buffers so they shall be set together. it means less room for error on the user's side.

Are there any uses cases when the render mode is changed but the buffers are not?

Reply:
I thought from name of the API `lv_display_set_draw_buffers`, it should only update draw buffers. Render mode is kind of independent of buffer. For example, if the buffer is full screen sized, user can choose to use all render mode freely.

There's use case in SDL when window resizes, the draw buffer is updated but not render mode.

Reply:
I see your point, but I'm afraid a lot of users will forget to set the render mode... 

Let's keep them separated, and show it in all examples and docs. It should be enough too.